tests/Domain/GetSongsListTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+require_once '../../src/Domain/GetSongsList.php';
+
+function testGetSongsList() {
```

Comment:
Please make sure to add a return type declaration. The curly brace should be on its own line (https://www.php-fig.org/psr/psr-12/#44-methods-and-functions).